tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -520,6 +542,50 @@ def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name,
     assert_that(file_permissions).is_equal_to("0100777")
 
 
+def _test_storage_capacity(remote_command_executor, mount_dir, storage_capacity):
+    logging.info("Test FSx storage capacity")
+    if storage_capacity:
```

Comment:
Can we remove these? These conditions will fail silently if we decreased coverage by accident

Please double check and change for all other assertions as well

Reply:
Done